src/tasks/selectors.js

```diff
@@ -62,5 +63,16 @@ export const taskFilters = {
         return ((task.assignee && (task.assignee.id === auth)) ||
           (task.creator && task.creator.id === auth));
       });
-  }
+  },
+
+  query: (tasks, filter) => {
```

Comment:
`filter.text` already init as empty string, so the check is redundant.
`filter` function will return the `tasks` array if nothing found so this selector can be:

```JS
query: (tasks, filter) =>
    tasks.filter(task => {
      const filterText = filter.text.toLowerCase();
      const taskTitle = task.title.toLowerCase();
      const taskDescription = task.description.toLowerCase();

      return taskTitle.includes(filterText) || taskDescription.includes(filterText);
})
```

Reply:
Actually filter.text is failing when I remove the check for null